recipes-python/pyqt5/python3-pyqt5_5.11.3.bb

```diff
@@ -0,0 +1,7 @@
+DEPENDS = "sip3 sip3-native qtbase python3"
+
+inherit qmake5 python3native python3-dir distro_features_check
```

Comment:
This qmake5, distro_features_check inherit depends in the inc file together with qtbase dependency and RDEPENDS.

And it would make sense to move the .inc file creation to the previous commit, because it's already using .inc file which doesn't exist (or squash them together).